### ISSUE TITLE
Unable to start watch - port 8384 in use

### ISSUE THREAD

**Issue by xAt0mZ** [vapor-ware/ksync#311]
Hello
I installed a new minikube cluster and ksync.
ksync init went fine, ksync doctor only tells me I need to start the watch, so did I.

But I'm unable to do it, getting the following error...
```
FATA[0001] Something is running on 8384 (run 'lsof -i :8384' to find out). Please stop that process before continuing.
```

`lsof -i :8384` returns nothing so I don't know what to do now.
Already cleaned ksync and recreated the k8s cluster, without improvement
Any solution ?

![2019-09-05-204653_1087x487_scrot](https://user-images.githubusercontent.com/7642331/64370301-7ad6bb00-d01e-11e9-96d8-5aed723260e7.png)

**Comment by xAt0mZ** [comment]
Found the solution !

1. Found that `syncthing` processes were still running on my laptop.
2. SIGKILLed them all with htop (that's rude, sorry for this).
3. Started syncthing by myseft to get some logs of why it was failing.

Problem was the following : syncthing failed to start (unable to find localhost on 8.8.8.8:53) but process kept running on background after failure, even when the process wasn't running anymore in my shell... :upside_down_face: 

Solved by adding `127.0.0.1 localhost` in `/etc/hosts` and restarting ksync :smile:

**Comment by grampelberg** [comment]
Good call! I've had a tough time trying to get go to manage processes correctly.
